public/app/features/explore/TableContainer.tsx

```diff
@@ -43,6 +43,13 @@ export class TableContainer extends PureComponent<TableContainerProps> {
 
     return (
       <Collapse label="Table" loading={loading} collapsible isOpen={showingTable} onToggle={this.onClickTableButton}>
+        {!tableResult?.length && (
+          <div className="logs-panel-meta">
+            <div className="logs-panel-meta__item">
+              <span className="logs-panel-meta__value">0 series returned</span>
+            </div>
+          </div>
```

Comment:
this seems a bit hacky to use logs panel styles in the table

Reply:
Yeah kinda breaks the BEM abstraction. What about creating a separate `MetaInfoText` component or something like that that would contain the styles. I think that is the way to share styles when using css-in-js solutions for styling.

Reply:
@aocenas great idea with separating it to another component. On it.